content/api/commands/getalllocalstorage.md

```diff
@@ -0,0 +1,93 @@
+---
+title: getAllLocalStorage
+---
+
+Get data in localStorage for all origins with which the test has interacted.
+
+## Syntax
+
+```javascript
+cy.getAllLocalStorage()
+cy.getAllLocalStorage(options)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+cy.getAllLocalStorage()
+```
+
+### Arguments
+
+**<Icon name="angle-right"></Icon> options** **_(Object)_**
+
+Pass in an options object to change the default behavior of
+`cy.getAllLocalStorage()`.
+
+| Option | Default | Description                                                                              |
+| ------ | ------- | ---------------------------------------------------------------------------------------- |
+| `log`  | `true`  | Displays the command in the [Command log](/guides/core-concepts/cypress-app#Command-Log) |
+
+### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
+
+`cy.getAllLocalStorage()` yields an object where the keys are origins and the
+values are key-value pairs of localStorage data.
```

Comment:
```suggestion
values are key-value pairs of `localStorage` data.
```

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/4876/commits/1b31d3d09ba10bea4afcd9cdc34af0fd7480bd79